queries.sql

```diff
@@ -7,4 +7,49 @@ SELECT date_of_birth from animals WHERE name='Agumon' OR name='Pikachu';
 SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
-SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT name from animals WHERE name iLIKE '%mon';
+SELECT * from animals WHERE EXTRACT(year from date_of_birth) BETWEEN 2016 AND 2019;
+SELECT name from animals WHERE neutered=true AND escape_attempts < 3;
+SELECT date_of_birth from animals WHERE name='Agumon' OR name='Pikachu';
+SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
+SELECT * from animals WHERE neutered = true;
+SELECT * from animals WHERE name != 'Gabumon';
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+begin;
+update animals set species='unspecified';
+rollback;
+
+begin;
+
+update animals SET species = 'Degimon' where name ilike '%mon';
+UPDATE animals SET species = 'Pokemon' where name not ilike '%mon';
+commit;
+
+begin;
+
+DELETE FROM animals;
+rollback;
+
+begin;
+savepoint aceUltra;
+DELETE from animals where date_of_birth >= '2022-01-01';
+rollback to aceUltra;
+
+savepoint luffy;
+
+UPDATE animals SET weight_kg = weight_kg * -1;
+rollback to luffy;
+UPDATE animals SET weight_kg = weight_kg * -1 where weight_kg < 0;
+commit;
+
+SELECT count (*) AS animalCount from animals;
+SELECT * from animals where escape_attempts < 1;
```

Comment:
- [ ] I think it would be better if you update this query to correctly answer this question **How many animals have never tried to escape?**(i.e the answer is supposed to be a number)
e.g.:
```PostgreSQL
 .... COUNT(*) FROM animals WHERE escape_attempts = 0;
```